src/s1b_opening_crawl.tw2

```diff
@@ -0,0 +1,15 @@
+::Opening crawl
+
+The year is 20XX. Wicked clown shit is the new law.
+
+(click: "the new law") [(transition: "dissolve") +(transition-time: 3s)[Years ago, the annual Gathering of the Juggalos festival ran long. Law enforcement’s response was swift and brutal. But the Juggalos held their ground.]]
```

Comment:
One little nitpick: I don't like how links go from bold text to plain text once you click them using this method. However, this `click` method is _super_ handy, so this is actually a critique of our default text styles. I'd love to see us get rid of the default font and bolded links. :)

Reply:
Absolutely agree. It looks really rough and it's definitely something I want to fix in styling so that we can freely use this click/transition method in more places.

Reply:
yeah I think the bold links convention is a Harlowe thing? either way I've usually CSS'ed that formatting away when I've done other Twine projects. The thing I hate the most about it is how once it goes un-bold, all the text shifts around into new line breaks & such. >:(

Reply:
The text shift suuuuucks for sure. I'd love to just style links as a diff color instead of bolding. I'd like to reserve bolding for emphasis instead of indicating you can click.